src/redux/features/appVersion/appVersionThunks.ts

```diff
@@ -0,0 +1,15 @@
+import { Dispatch } from 'redux';
+import { getLatestAppVersion } from 'src/api/appVersion';
+import { RootState } from 'src/redux/rootReducer';
+import { setVersionAction } from './appVersionSlice';
+
+export const requestAppVersion = () => {
+  return async (dispatch: Dispatch, _getState: () => RootState) => {
+    try {
+      const response = await getLatestAppVersion();
+      dispatch(setVersionAction(response.version));
+    } catch (e) {
+      // add error handling
```

Comment:
TODO: add some error handling actions here. I am unsure what the behavior should be at this time.

also TODO: show referencing the root state within this request as well as what it looks like to pass in parameters

Reply:
let it be for now, maybe a console.error (with eslint disable no-console).. we don't want to surface this app version error to user.